requirements.txt

```diff
@@ -2,4 +2,4 @@ pyyaml
 argparse
 logging
 pyyaml
-discord
\ No newline at end of file
+discord
```

Comment:
These all need to have explicit versions defined.

This command will give you the specific discord version and you can specify it in the reqs file like discord==1.2.3.
```pip3 freeze | grep discord```